File: Android/CrimsonClubs/app/src/main/java/crimsonclubs/uacs/android/crimsonclubs/UserDto.java
package crimsonclubs.uacs.android.crimsonclubs;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;


public class UserDto {

    @Expose
    @SerializedName("id")
    public int id;
    @Expose
    @SerializedName("first")
    public String first;
    @Expose
    @SerializedName("last")
    public String last;
    @Expose
    @SerializedName("isOrganizationAdmin")
    public boolean isOrganizationAdmin;
    @Expose
    @SerializedName("organizationId")
    public int organizationId;

}

File: CrimsonClubs/CrimsonClubs.Models/Dtos/UserDto.cs
using CrimsonClubs.Models.Entities;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace CrimsonClubs.Models.Dtos
{
    public class UserDto
    {
        public int Id { get; set; }
        public string Email { get; set; }
        public string First { get; set; }
        public string Last { get; set; }
        public bool IsOrganizationAdmin { get; set; }
        public int OrganizationId { get; set; }

        public UserDto()
        {

        }

        public UserDto(User dbo)
        {
            Id = dbo.Id;
            Email = dbo.Email;
            First = dbo.First;
            Last = dbo.Last;
            IsOrganizationAdmin = dbo.IsOrganizationAdmin;
            OrganizationId = dbo.OrganizationId;
        }
    }
}
